### PR TITLE
Feature/1269 sections api update

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by potehinre — APPROVED** [review]

File: packages/api/sections.go [hunk 1]
```diff
@@ -100,6 +100,21 @@ func getSectionsHandler(w http.ResponseWriter, r *http.Request) {
...
+			roles.SetTablePrefix("1")
+			role, err := roles.Get(nil, client.RoleID)
+
+			if err != nil {
+				logger.WithFields(log.Fields{"type": consts.DBError, "error": err, "table": table}).Error("Getting role by id")
+				errorResponse(w, err)
+				return
+			}
+			if role == true {
+				item["default_page"] = roles.DefaultPage
```
Comment (dchertkov) [inline packages/api/sections.go]: There is no condition for blank page of role

File: packages/api/sections.go [hunk 2]
```diff
@@ -100,6 +100,21 @@ func getSectionsHandler(w http.ResponseWriter, r *http.Request) {
 			}
 		}
 
+		if item["status"] == "2" {
```
Comment (dchertkov) [inline packages/api/sections.go]: Magic string

Comment (dchertkov) [inline packages/api/sections.go]: Magic string

File: packages/api/sections.go (hunk 1 above)
Comment (dchertkov) [inline packages/api/sections.go:127]: There is no condition for blank page of role

**Review by dchertkov — CHANGES_REQUESTED** [review]

**Review by potehinre — APPROVED** [review]

**Pull request merged**